hack/generate/samples/internal/ansible/advanced_molecule.go

```diff
@@ -61,7 +61,7 @@ func (ma *AdvancedMolecule) Prepare() {
 func (ma *AdvancedMolecule) Run() {
 	log.Infof("creating the project")
 	err := ma.ctx.Init(
-		"--plugins", "ansible",
+		"--plugins", "ansible,manifests,scorecard",
```

Comment:
We cannot do it. Any SDK user who builts a project with the tool will scaffold/use these both plugins by default. 
In the future, it will have optional plugins as well. However, these both require to be the default behaviour.

Reply:
I am aware of this, and it has to be solved before this PR is able to be merged.

Reply:
Solved